### ISSUE TITLE
Unfortunately, language code hardcoded to "en" and no option to change it.

### ISSUE THREAD

**Issue by belonesox** [akhildevelops/summarizer#23]
Unfortunately, language code hardcoded to "en" and no option to change it.

**Comment by akhildevelops** [comment]
Yup, didn't thought of generalizing it as I was focused on english transcripts, would be nice if you can contribute to address this issue.

**Comment by akhildevelops** [comment]
https://github.com/akhildevelops/summarizer/pull/25
